### PR TITLE
enh(UI): Handle range thresholds on widgets

### PR DESCRIPTION
**Pull request by Thebarda**
## Description

This handles range thresholds on widgets

## Type of change

- [ ] Patch fixing an issue (non-breaking change)
- [ ] New functionality (non-breaking change)
- [ ] Breaking change (patch or feature) that might cause side effects breaking part of the Software

## Target serie

- [ ] 21.10.x
- [ ] 22.04.x
- [ ] 22.10.x
- [ ] 23.04.x
- [x] 23.10.x (master)

<h2> How this pull request can be tested ? </h2>

Please describe the **procedure** to verify that the goal of the PR is matched. Provide clear instructions so that it can be **correctly tested**.

Any **relevant details** of the configuration to perform the test should be added.

#### Community contributors & Centreon team

- [x] I have followed the **coding style guidelines** provided by Centreon
- [ ] I have commented my code, especially new **classes**, **functions** or any **legacy code** modified. (***docblock***)
- [ ] I have commented my code, especially **hard-to-understand areas** of the PR.
- [x] I have **rebased** my development branch on the base branch (master, maintenance).

### CODE REVIEW

File: centreon/lang/de_DE.UTF-8/LC_MESSAGES/messages.po [hunk 1]
```diff
@@ -15919,6 +15919,36 @@ msgstr ""
...
+# msgid "Last 12 months"
+# msgstr ""
+
+# msgid "Start"
+# msgstr ""
+
+# msgid "End"
+# msgstr ""
+
+# msgid "Value customized"
```
Comment (smau-centreon) [inline centreon/lang/de_DE.UTF-8/LC_MESSAGES/messages.po:15949]:
```suggestion
# msgid "Custom value"
```

File: centreon/lang/es_ES.UTF-8/LC_MESSAGES/messages.po [hunk 2]
```diff
@@ -16253,6 +16253,36 @@ msgstr ""
...
+# msgid "Last 12 months"
+# msgstr ""
+
+# msgid "Start"
+# msgstr ""
+
+# msgid "End"
+# msgstr ""
+
+# msgid "Value customized"
```
Comment (smau-centreon) [inline centreon/lang/es_ES.UTF-8/LC_MESSAGES/messages.po:16283]:
```suggestion
# msgid "Custom value"
```

File: centreon/lang/fr_FR.UTF-8/LC_MESSAGES/messages.po [hunk 3]
```diff
@@ -7173,13 +7160,6 @@ msgstr "Les 14 précédents jours"
 msgid "Last 28 Days"
 msgstr "Les 28 précédents jours"
```
Comment (smau-centreon) [inline centreon/lang/fr_FR.UTF-8/LC_MESSAGES/messages.po:7161]:
```suggestion
msgstr "Les 28 derniers jours"
```

File: centreon/lang/pt_BR.UTF-8/LC_MESSAGES/messages.po [hunk 4]
```diff
@@ -16718,6 +16718,36 @@ msgstr ""
...
+# msgid "Last 12 months"
+# msgstr ""
+
+# msgid "Start"
+# msgstr ""
+
+# msgid "End"
+# msgstr ""
+
+# msgid "Value customized"
```
Comment (smau-centreon) [inline centreon/lang/pt_BR.UTF-8/LC_MESSAGES/messages.po:16748]:
```suggestion
# msgid "Custom value"
```

File: centreon/lang/pt_PT.UTF-8/LC_MESSAGES/messages.po [hunk 5]
```diff
@@ -16707,6 +16707,36 @@ msgstr "Você não salvou as alterações, deseja continuar?"
...
+# msgid "Last 12 months"
+# msgstr ""
+
+# msgid "Start"
+# msgstr ""
+
+# msgid "End"
+# msgstr ""
+
+# msgid "Value customized"
```
Comment (smau-centreon) [inline centreon/lang/pt_PT.UTF-8/LC_MESSAGES/messages.po:16737]:
```suggestion
# msgid "Custom value"
```

**Review by smau-centreon — CHANGES_REQUESTED** [review]

File: centreon/lang/fr_FR.UTF-8/LC_MESSAGES/messages.po [hunk 6]
```diff
@@ -7128,12 +7128,6 @@ msgstr "Les 6 dernières heures"
 msgid "Last 12 Hours"
```
Comment (smau-centreon) [inline centreon/lang/fr_FR.UTF-8/LC_MESSAGES/messages.po:7128]:
```suggestion
msgid "Last 12 hours"
```

File: centreon/lang/fr_FR.UTF-8/LC_MESSAGES/messages.po [hunk 7]
```diff
@@ -7154,13 +7148,6 @@ msgstr "Les 4 précédents jours"
 msgid "Last 5 Days"
```
Comment (smau-centreon) [inline centreon/lang/fr_FR.UTF-8/LC_MESSAGES/messages.po:7148]: ### 
```suggestion
msgid "Last 5 days"
```

File: centreon/lang/fr_FR.UTF-8/LC_MESSAGES/messages.po [hunk 8]
```diff
@@ -7171,14 +7158,7 @@ msgstr "Les 14 précédents jours"
 #: centreon-web/www/include/views/graphTemplates/formGraphTemplate.php:110
 #: centreon-web/www/include/eventLogs/viewLog.php:176
 msgid "Last 28 Days"
```
Comment (smau-centreon) [inline centreon/lang/fr_FR.UTF-8/LC_MESSAGES/messages.po:7160]:
```suggestion
msgid "Last 28 days"
```

**Review by smau-centreon — CHANGES_REQUESTED** [review]

**Review by SoldierBoy31 — APPROVED** [review]

**Review by jeremyjaouen — APPROVED** [review]

**Review by smau-centreon — APPROVED** [review]

**Pull request merged**